spec/index.html

```diff
@@ -465,6 +465,7 @@ <h6 class="defcap">Definition of a Simple Interpretation</h6>
             <li>A mapping <em>IEXT</em> from <em>IP</em> into the powerset of <em>IR</em> x <em>IR</em>, i.e., the set of sets of pairs <em>&lt;x, y&gt;</em> with <em>x</em> and <em>y</em> in <em>IR</em>.</li>
             <li>A mapping <em>IS</em> from <em>IRIs</em> into (<em>IR</em> union <em>IP</em>).</li>
             <li>A partial mapping <em>IL</em> from literals into <em>IR</em>.</li>
+            <li><span style="background-color: Cyan;color: Black">6. An injective mapping RE from IR x IP x IR into IR, called the denotation of triple terms</span> </li>
```

Comment:
```suggestion
            <li><span style="background-color: Cyan;color: Black">An injective mapping RE from IR x IP x IR into IR, called the denotation of triple terms</span> </li>
```
Remove number - that is generated by the `<li>`.